R/ltcad.R

```diff
@@ -68,6 +61,55 @@ epidist_family.epidist_ltcad <- function(data, family = "lognormal") {
   )
 }
 
+#' Define priors for the LTCAD model
+#'
+#' @method epidist_priors epidist_ltcad
+#' @family ltcad
+#' @export
+epidist_priors.epidist_ltcad <- function(priors_list, data,
+                                         family = epidist_family(data),
```

Comment:
Possibly we want to change things so that the user can pass in priors like `function(prior1, prior2, prior3, data, family)` then we put them into a list